listeners/onEnvStart.php

```diff
@@ -0,0 +1,18 @@
+<?php
+require "./services/api.php";
+
+function run($props, $event, $api)
+{
+    $res = executeQuery($api, "counter", array(
+        "user" => "global"
+    ));
+
+    $counters = json_decode($res->getBody());
```

Comment:
You should decode the JSON in the `executeQuery` method:

```suggestion
    $counters = executeQuery($api, "counter", array(
        "user" => "global"
    ));
```